cfme/tests/configure/test_access_control.py

```diff
@@ -242,6 +246,22 @@ def tenant_custom_role(appliance, request, provider):
     tenant_role.delete_if_exists()
 
 
+@pytest.fixture()
+def custom_user(appliance):
+    group = appliance.collections.groups.instantiate("EvmGroup-user_self_service")
+    user = appliance.collections.users.create(
```

Comment:
Why don't you use REST API to create the user? It would make this thing faster.

Reply:
Done